tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/ScopedBuilder.scala

```diff
@@ -49,6 +49,10 @@ class ScopedBuilder
    */
   def useTrustedEx(ex: TlaEx): TBuilderInstruction = ex.point[TBuilderInternalState]
 
+  ////////////////////
+  // HYBRID METHODS //
```

Comment:
In which sense are they hybrid?

Reply:
They are defined using other builder methods, typically from 2+ sub-builders (e.g. `eql` from `Base` and `prime` from `Action`)